config/migrations/20221007130000-case-and-decisionmaking-publication-governmentareas-migration-kanselarij.sparql

```diff
@@ -0,0 +1,15 @@
+PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
+PREFIX pub: <http://mu.semte.ch/vocabularies/ext/publicatie/>
+WITH <http://mu.semte.ch/graphs/organizations/kanselarij>
+INSERT {
+  ?publicationFlow pub:beleidsveld ?decisionGovernmentArea .
+}
+WHERE {
+   ?publicationFlow a pub:Publicatieaangelegenheid .
+   ?publicationFlow dossier:behandelt ?case .
+   ?case dossier:Dossier.isNeerslagVan ?decFlow .
+   ?decFlow dossier:doorloopt ?subcase .
+    ?subcase besluitvorming:beleidsveld ?decisionGovernmentArea .
+   FILTER NOT EXISTS { ?publicationFlow pub:beleidsveld ?decisionGovernmentArea }
```

Comment:
Die `FILTER NOT EXISTS` lijkt mij overbodig?  'Worst case' wordt een bestaande triple nog eens geinsert. 't Is niet dat we dan met 2 triples gaan eindigen.